ext/mbstring/tests/mb_str_unknown_encoding.phpt

```diff
@@ -11,7 +11,6 @@ mb_chr(1, 'UTF-0');
 mb_convert_case('coudenys', MB_CASE_UPPER, 'UTF-0');
 mb_convert_encoding('coudenys', 'UTF-8', 'UTF-0');
 mb_convert_kana('coudenys', 'KV', 'UTF-0');
-mb_decode_numericentity('coudenys', 'KV', 'UTF-0');
```

Comment:
This test should stay, just with `[]` instead of `'KV'`. It just checks for the encoding error...

Reply:
Thanks! Done.